src/test/scala/nl.knaw.dans.easy.deposit/StagedFilesSpec.scala

```diff
@@ -17,26 +17,25 @@ package nl.knaw.dans.easy.deposit
 
 import java.net.{ URL, UnknownHostException }
 import java.nio.file.Paths
-import java.util.UUID
 
 import better.files.StringExtensions
 import nl.knaw.dans.bag.v0.DansV0Bag
 import nl.knaw.dans.lib.error._
 
 import scala.util.Success
 
-class StagedFilesTargetSpec extends TestSupportFixture {
+class StagedFilesSpec extends TestSupportFixture {
 
   private val draftDir = testDir / "draft"
-  private val stagedDir = testDir / "stage-for-submit"
+  private val stagedDir = testDir / "staged"
 
   override def beforeEach(): Unit = {
     super.beforeEach()
     clearTestDir()
     stagedDir.createDirectories()
   }
 
-  "moveAllFrom" should "add payload files" in {
+  "DataFiles.moveAll" should "add payload files" in {
```

Comment:
these tests should be moved to `DataFilesSpec` to mimic the class structure in `src/main`